### ISSUE TITLE
Move this package as an extension? 

### ISSUE THREAD

**Issue by longemen3000** [longemen3000/ForwardDiffOverMeasurements.jl#3]
This package was created before the existence of Julia 1.9 extensions

**Comment by gdalle** [comment]
It's rather easy to do (I can give a hand if needed), the only question is where to put the extension. I would say Measurements?

**Comment by dalum** [comment]
Recently ran into the ambiguity issues with:
```
/(::ForwardDiff.Dual, ::Measurements.Measurement)
```
but I'm a bit hesitant to add this package as a dependency (since it's doing high piracy). Would anything be needed besides just copying the contents over into an extension module in Measurements.jl?

**Comment by longemen3000** [comment]
Yes, this package is basically an extension, as it does not add any methods. Maybe i should open a PR on Measurements...

The main blocker was that not all Measurements.jl functions are supported (mainly the constructors, anf getters/setters), but the rest works.

**Comment by longemen3000** [comment]
I think i'm gonna do the following strategy:
1. We check if there is support for `Base.get_extension`
1. We check if there is a Measurements.jl extension named `:MeasurementsForwardDiffExt` 
1. If the extension exists, do not load the code
 
if any of the cases fails, we load the extension code. That should work for older julia versions and older Measurements.jl versions.
